pkg/util/helper/predicate.go

```diff
@@ -34,16 +34,9 @@ import (
 // - execution controller working in karmada-controller-manager
 // - work status controller working in karmada-controller-manager
 func NewExecutionPredicate(mgr controllerruntime.Manager) predicate.Funcs {
```

Comment:
1. Can we give this function a meaningful name now? As it only filters clusters with `Push Mode`.